FreeRTOS/Test/CMock/smp/single_priority_no_timeslice_covg/single_priority_no_timeslice_covg_utest.c

```diff
@@ -413,3 +413,123 @@ void test_task_step_tick_xNextTaskUnblockTime_not_equal( void )
     xTickCount = 0U;
     vTaskStepTick((TickType_t)10U);
 }
+
+/**
+ * @brief AWS_IoT-FreeRTOS_SMP_TC-<TBD>
```

Comment:
This tag refers to a specific test case in JAMA, I dont think we need this for coverage tests. Maybe a new identifier specifically for coverage tests? That way it will be clear in any doxygen output the difference between a functional test and a coverage unit test (especially if they are in the same C file)

Reply:
Agree with the suggestion. This will be updated in this PR.